server/api/src/rsModAVUMetadata.cpp

```diff
@@ -146,81 +144,106 @@ _rsModAVUMetadata( rsComm_t *rsComm, modAVUMetadataInp_t *modAVUMetadataInp ) {
     }
 
     if ( strcmp( modAVUMetadataInp->arg0, "add" ) == 0 ) {
-        status = chlAddAVUMetadata( rsComm, 0,
-                                    modAVUMetadataInp->arg1,
-                                    modAVUMetadataInp->arg2,
-                                    modAVUMetadataInp->arg3,
-                                    modAVUMetadataInp->arg4,
-                                    modAVUMetadataInp->arg5 );
+        status = chlAddAVUMetadata(rsComm,
+                                   modAVUMetadataInp->arg1,
+                                   modAVUMetadataInp->arg2,
+                                   modAVUMetadataInp->arg3,
+                                   modAVUMetadataInp->arg4,
+                                   modAVUMetadataInp->arg5,
+                                   &modAVUMetadataInp->condInput);
     }
     else if ( strcmp( modAVUMetadataInp->arg0, "adda" ) == 0 ) {
```

Comment:
We should definitely deprecate this command in 4.3.x. Sheesh.

Reply:
oh, yeah, please make an issue for that.     ew.

Reply:
i agree. it is covered by the admin option now.

@trel any thoughts on this?

Reply:
deprecate.

Reply:
let's handle the deprecation in a separate PR.

Reply:
yes

Reply:
issue created: #6187